server/controllers/chartsController.js

```diff
@@ -209,3 +210,10 @@ function calculateStudySectionTargetValue(
 
   return studySectionTotalCount || 0
 }
+
+function isNoValueCounted(subjectDayData, value, variable) {
+  if (!subjectDayData.length > 0) return false
+  return subjectDayData.every((day) => {
+    return day[variable] === value
+  })
```

Comment:
suggesting this to slightly rework the logic but I need you to make sure this is actually what we want!

```suggestion
  if (subjectDayData.length === 0) return true

  return subjectDayData.every((day) => day[variable] === value)
```

Reply:
thank you for pointing this out, waiting to hear back wether an empty subjectDayData is counted towards the no value

Reply:
I think we can run with the assumption that it is for now. If you think about it, not having any data for a subject in a study is basically the same thing as having days of data without anything for the variable.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/199/commits/d0d2edab5683a1cfac5339f1fbabf3a97a2213a3